av/video/frame.pyi

```diff
@@ -19,34 +28,37 @@ class PictureType(EnumItem):
 
 class VideoFrame(Frame):
     format: VideoFormat
-    pts: int
-    time: float
```

Comment:
This should remain because pts and time are always not None in VideoFrame

Reply:
Thanks, resolved!